src/core/lombok/bytecode/PoolConstantsApp.java

```diff
@@ -66,7 +66,7 @@
 		CmdReader<CmdArgs> reader = CmdReader.of(CmdArgs.class);
 		CmdArgs args;
 		try {
-			args = reader.make(raw.toArray(new String[0]));
+			args = reader.make(raw.toArray(new String[raw.size()]));
```

Comment:
Hi,

in fact, this is a performance drop 
https://gist.github.com/twillouer/ac13eb1dadc8a270f821
and http://cr.openjdk.java.net/~shade/scratch/ToArrayBench.java